stable/traefik/templates/configmap.yaml

```diff
@@ -23,9 +23,14 @@ data:
       [entryPoints.http]
       address = ":80"
       compress = {{ .Values.gzip.enabled }}
-        {{- if and .Values.ssl.enabled .Values.ssl.enforced }}
+        {{- if .Values.ssl.enforced }}
         [entryPoints.http.redirect]
+          {{- if .Values.ssl.enabled }}
           entryPoint = "https"
+          {{- else }}
+          regex = "^http://(.*)"
+          replacement = "https://$1"
```

Comment:
Is there any reason to not just always use the replacement above instead of sometimes using `entryPoint = "https"`?

Reply:
Maybe it performs better, but I don't know the internals of Traefik to prove this statement. At least that's what Traefik is using in their own documentation for HTTPS redirects.

Reply:
It is a redirect either way. Unless there's some real justification we can uncover, eliminating that conditional will just make things simpler here.